tests/helm_test.go

```diff
@@ -0,0 +1,24 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/aks-periscope/pkg/utils"
+)
+
+func TestHelm(t *testing.T) {
+
+	output, err := utils.RunCommandOnContainer("helm", "list", "--all-namespaces")
```

Comment:
Is there a way/an ask for you to write a test where you create a helm collector and then run `Collect()` on it, this would be the most comprehensive form of testing

Reply:
from a discussion @sophsoph321 had out of band:
When I tested helmCollector.Collect in an isolated unit, I was getting this error: Fail to run command on host: exec: "nsenter": executable file not found in %PATH%

We should look to make the dependency on nsenter injectable or able to be abstracted away for unit testing purposes